app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -42,14 +43,17 @@
 import butterknife.OnClick;
 
 public class PresentationActivity extends StandardActivity
-        implements TextToSpeech.OnInitListener, ColorChooserDialog.ColorCallback {
+        implements ColorChooserDialog.ColorCallback, TextToSpeechManager.Listener {
 
     public static final String LIST_NAME_KEY = "listName";
     public static final String DRUMROLL_FILE_NAME = "drumroll.mp3";
 
-    @BindView(R.id.header) TextView header;
-    @BindView(R.id.names) TextView names;
-    @BindColor(R.color.dark_gray) int darkGray;
+    @BindView(R.id.header)
```

Comment:
nit: Can you undo these formatting changes? It makes it hard to figure out which bind corresponds to which variable.